### ISSUE TITLE
getting parse error 150 randomly when using pfimageview

### ISSUE THREAD

**Issue by tzapu** [ParsePlatform/ParseUI-iOS#103]
Hello,

I have a table cell that has 3 pfimageviews  and some text. I get random errors on random pictures at times with parse error code 150. i reload the table and everything is fine, images that erred earlier load just fine .

Tried both loadInBackground and loadInBackground:completion to no avail.
```
//[thumbView loadInBackground];
            //[noteThumbView loadInBackground];
            //[userThumbView loadInBackground];
    
           [thumbView loadInBackground:^(UIImage *image, NSError *error) {
                if (error) {
                    DLog(@"error thumb %@", error);
                }
                [noteThumbView loadInBackground:^(UIImage *image, NSError *error) {
                    if (error) {
                        DLog(@"error loading note thumb %@", error);
                    }
                    [userThumbView loadInBackground:^(UIImage *image, NSError *error) {
                        if (error) {
                            DLog(@"error loading user thumb %@", error);
                        }
                    }];
                }];
            }];
```

error
```
error loading user thumb Error Domain=Parse Code=150 "The operation couldn’t be completed. (Parse error 150.)"
```

**Comment by damianesteban** [comment]
Ive been having the same issue for months.  I don't think this is functioning properly.

**Comment by ericcgu** [comment]
Yes

**Comment by tzapu** [comment]
still happening
ViewController tableView:cellForRowAtIndexPath:]_block_invoke_2 [Line 228] error loading note thumb Error Domain=Parse Code=150 "The operation couldn’t be completed. (Parse error 150.)"
Jun 25 12:56:37 [73362] <Error>: ImageIO: JPEG Corrupt JPEG data: 443453 extraneous bytes before marker 0xd3
Jun 25 12:56:37 [73362] <Error>: ImageIO: JPEG JPEG datastream contains no image
Jun 25 12:56:37 [73362] <Error>: ImageIO: JPEG Corrupt JPEG data: 443453 extraneous bytes before marker 0xd3
Jun 25 12:56:37 [73362] <Error>: ImageIO: JPEG JPEG datastream contains no image

image is valid for sure, and a reload of the view controller shows it ok

**Comment by tzapu** [comment]
this seems to happen when loading from parse multiple copies of the same image at the same time

**Comment by tzapu** [comment]
this has been recognized as a bug and assigned
https://developers.facebook.com/bugs/1593418017585752

**Comment by nlutsenko** [comment]
Sorry for wait time on this one.
It is indeed a valid bug, and it looks to be the one inside Parse SDK itself, not ParseUI.
Looking into it, will update the external bug and this issue when it's resolved.

**Comment by nlutsenko** [comment]
Fixed in Parse SDK [1.8.0](https://github.com/ParsePlatform/Parse-SDK-iOS-OSX/releases)
